app/src/gecko/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -169,8 +169,8 @@ private void updateBlocking() {
             }
         }
 
-        private GeckoSession.ContentListener createContentListener() {
-            return new GeckoSession.ContentListener() {
+        private GeckoSession.ContentDelegate createContentDelegate() {
```

Comment:
\[PMD\] INFO: The method 'createContentDelegate' has a Standard Cyclomatic Complexity of 11\.